babel.config.cjs

```diff
@@ -14,7 +14,14 @@ module.exports = {
                 modules: process.env.NODE_ENV === "test" ? "commonjs" : false,
             },
         ],
-        "@babel/preset-typescript",
+        [
+            "@babel/preset-typescript",
+            {
+                // Node.js requires .js extensions on ESM imports. Our tests, however, still
+                // run in CommonJS (see above).
```

Comment:
```suggestion
                // Node.js requires .js extensions on ESM imports. However, `jest` uses the TS source (via babel)
                // and fails to resolve the `.js` names. To resolve this, we use the `.ts` names in the source, and
                // rewrite the extensions for `.js` during transpilation, *except* when we are targetting jest.
```

Reply:
Sorry yes, I was being sloppy there. This is much better.